### PR TITLE
Milestone title should be unique #2256

### PR DESCRIPTION
**Pull request by joobisb**
### Before submitting a pull-request to GitBucket I have first:

- [x] read the [contribution guidelines](https://github.com/gitbucket/gitbucket/blob/master/.github/CONTRIBUTING.md)
- [x] rebased my branch over master
- [x] verified that project is compiling
- [x] verified that tests are passing
- [x] squashed my commits as appropriate *(keep several commits if it is relevant to understand the PR)*
- [x] [marked as closed using commit message](https://help.github.com/articles/closing-issues-via-commit-messages/) all issue ID that this PR should correct

### CODE REVIEW

**Comment by joobisb** [comment]
@takezoe

File: src/main/scala/gitbucket/core/util/Validations.scala [hunk 1]
```diff
@@ -45,6 +45,20 @@ trait Validations {
       }
   }
 
+  /**
+   * Constraint for the milestone identifier.
+   */
+  def milestone: Constraint = new Constraint() {
```
Comment (takezoe) [inline src/main/scala/gitbucket/core/util/Validations.scala]: Why do you add this constraint to the milestone? Also, if this constraint is used by only `milestoneForm`, it should be defined in `MilestonesController.scala` as same as `uniqueMilestone` constraint.

Comment (joobisb) [reply src/main/scala/gitbucket/core/util/Validations.scala]: @takezoe I just thought repository has this constraint and so I will add it to milestone as well. I will just remove this constraint.

**Comment by joobisb** [comment]
@takezoe I've removed the constraint. Please have a look.

**Pull request merged**